cgp/node_impl.py

```diff
@@ -11,35 +11,35 @@ class ConstantFloat(OperatorNode):
 
 
 class Add(OperatorNode):
-    """A node that adds its two inputs."""
+    """A node that adds its two addresses."""
```

Comment:
mhm. nope, i don't think we should rename this here ;) i mean we don't need to distort language, the important thing is to use good variables names in the code.